ipywidgets/widgets/widget_image.py

```diff
@@ -32,3 +33,40 @@ class Image(DOMWidget, ValueWidget, CoreWidget):
     width = CUnicode(help="Width of the image in pixels.").tag(sync=True)
     height = CUnicode(help="Height of the image in pixels.").tag(sync=True)
     value = Bytes(help="The image data as a byte string.").tag(sync=True)
+
+    @classmethod
+    def from_filename(cls, filename, **kwargs):
```

Comment:
I'd say `from_file`

Reply:
That's a reasonable change. The main reason I didn't go with `from_file` was that it seemed like `from_file` would take a stream buffer object (like an open `file`), rather than a `filename`.

I suppose I could change it so that it's `from_file` and try to infer whether it's already an open file or a file path.

Reply:
You're right, that will be expected, and a nice feature :)